Tests/Helper/Mocks/MockNetworkService2.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  MockNetworkService2.swift
+//  Hyperspace-iOS
+//
+//  Created by Adam Brzozowski on 1/31/18.
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+@testable import Hyperspace
+
+class MockNetworkService2: NetworkService {
+    var setup = false
```

Comment:
To make things more clear at the call site, you may want to consider modifying the name of this variable:
```
private(set) var initWithSessionNetworkActivityControllerWasCalled = false
```
Another option would be to take a somewhat less explicit name (that still gets the point across):
```
private(set) var defaultInitWasCalled = false
```

Reply:
Done